From: Unit 193 <unit193@ubuntu.com>
Date: Thu, 3 Jan 2019 21:12:03 +0100
Subject: use xscreensaver through the wrapper it ships

xscreensaver ships a wrapper that checks what DM is running,
and conditionally uses either dm-tool or gdmflexiserver.

Forwarded: not-needed
Last-Update: 2016-09-21
---
 scripts/xscreensaver.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/xscreensaver.desktop.in b/scripts/xscreensaver.desktop.in
index b18ba52..66d7dbf 100644
--- a/scripts/xscreensaver.desktop.in
+++ b/scripts/xscreensaver.desktop.in
@@ -4,6 +4,6 @@ Type=Application
 _Name=Screensaver
 _Comment=Launch screensaver and locker program
 Icon=preferences-desktop-screensaver
-Exec=xscreensaver -no-splash
+Exec=/usr/share/xscreensaver/xscreensaver-wrapper.sh -no-splash
 TryExec=xscreensaver
 OnlyShowIn=XFCE;
